src/main/java/frc/robot/commands/TeleopSwerve.java

```diff
@@ -38,16 +38,18 @@ public TeleopSwerve(Swerve s_Swerve, DoubleSupplier translationSup, DoubleSuppli
         this.strafeSup = strafeSup;
         this.rotationSup = rotationSup;
         this.robotCentricSup = robotCentricSup;
-        this.dampen = dampen;
-        this.speedDial = speedDial;
+        this.dampen = dampen.getAsBoolean();
+        this.speedDial = speedDial.getAsDouble();
+
+        
```

Comment:
You still need to store these as suppliers, otherwise the speed and dampen values will only be read when the swerve drive is initialized.